modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/import/new_import/import_layouts_resources.jsp

```diff
@@ -258,31 +258,33 @@ ManifestSummary manifestSummary = ExportImportHelperUtil.getManifestSummary(user
 														PortletDataHandlerControl[] importMetadataControls = portletDataHandler.getImportMetadataControls();
 														%>
 
-														<ul class="hide" id="<portlet:namespace />showChangeContent_<%= portlet.getRootPortletId() %>">
-															<li>
-																<span class="selected-labels" id="<portlet:namespace />selectedContent_<%= portlet.getRootPortletId() %>"></span>
-
-																<clay:button
-																	cssClass="content-link modify-link pr-1"
-																	data-portletid="<%= portlet.getRootPortletId() %>"
-																	data-portlettitle="<%= portletTitle %>"
-																	displayType="link"
-																	id='<%= liferayPortletResponse.getNamespace() + "contentLink_" + portlet.getRootPortletId() %>'
-																	label="change"
-																/>
-
-																<span id="<portlet:namespace />rightContentArrow_<%= portlet.getRootPortletId() %>">
-																	<clay:icon
-																		symbol="angle-right-small"
+														<c:if test="<%= !portlet.getRootPortletId().startsWith(ObjectPortletKeys.OBJECT_DEFINITIONS) %>">
```

Comment:
Not sure if this is the approach we want to use, because we are adding a dependency with Objects in the Export/Import. Let's discuss it internally